fdbclient/NativeAPI.actor.cpp

```diff
@@ -963,6 +963,20 @@ void setNetworkOption(FDBNetworkOptions::Option option, Optional<StringRef> valu
 				throw invalid_option_value();
 			}
 			break;
+		case FDBNetworkOptions::CLIENTBUGGIFY_ENABLE:
```

Comment:
Can we remove the `CLIENTBUGGIFY_DISABLE` option and simply default the `CLIENTBUGGIFY_ENABLE ` to false? This is totally non-blocking though.

Reply:
Good question - I don't know what makes more sense and I don't have strong opinions. My line of thinking was that you might want to be able to enable and disable client buggify multiple times during a single run. But that might be a rare enough use-case so that we might not want to have more options for this.

Reply:
I think I'm personally ok with having both options, but if anybody disagrees we should have that discussion now rather than try to remove the option from our API later.

Anybody else want to weigh in? @alecgrieser @dongxinEric @alexmiller-apple etc.